systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1328,6 +1335,148 @@ void testOwnerReferenceOfCASecrets() {
         });
     }
 
+    X509Certificate getCertificateFromSecret(Secret secret, String dataKey) {
+        String caCert = secret.getData().get(dataKey);
+        byte[] decoded = Base64.getDecoder().decode(caCert);
+        X509Certificate cacert = null;
+        try {
+            cacert = (X509Certificate)
+                    CertificateFactory.getInstance("X.509").generateCertificate(new ByteArrayInputStream(decoded));
+        } catch (CertificateException e) {
+            e.printStackTrace();
+        }
+        return cacert;
+    }
```

Comment:
I would move this to `SecretUtils` or end of this file, to make it more clear (for me it's clearer)

Reply:
Good idea, moved it there. Did not know about such class :-)